BaseFilter/sections/specific.txt

```diff
@@ -841,6 +841,9 @@ techarmor.xyz##.my-ad-container
 !
 ! NOTE: Specific rules
 !
+nanoreview.net##div[class^="card mb"][style^="position: relative;"]
+nanoreview.net##div.gtx_lol_o_desktop > .card > .pb.card-block
+nanoreview.net##article#the-app > div[data-nap-disclaimer] > .card-transparent
```

Comment:
I think these rules will be better. Please check them.
```suggestion
nanoreview.net###the-app > div[data-nap-disclaimer="exist"]
nanoreview.net##div[data-nap-atag^="soc-"] > div[class^="card"]
```

> div.gtx_lol_o_desktop > .card > .pb.card-block

I see the leftover is already blocked on my end.

Reply:
agree